api/src/main/java/de/adorsys/keymanagement/api/view/View.java

```diff
@@ -14,7 +19,23 @@
      */
     QueryResult<O> retrieve(String query);
 
+    /**
+     * Guarantees result uniqueness
+     * @param query understandable to implementation query.
+     * @return query result
+     */
     O uniqueResult(Q query);
+
+    /**
+     * Guarantees result uniqueness
+     * @param query understandable to implementation query as a string
+     * @return query result
+     */
     O uniqueResult(String query);
+
+    /**
+     * Retrieves all elements of target object
```

Comment:
Retrieves all elements (like aliases for `{@link ...AliasView}`) of target object

Reply:
done